### PR TITLE
Fix rtl issue for switch component

### PR DESCRIPTION
**Pull request by viktor-yang**
There is a rtl issue on the switch component, this PR is to fix it.

Before:
image
After:
image

Remember to include the following changes:

- [ ] `UNRELEASED.md` (See [How to write a good changelog entry](https://github.com/Skyscanner/backpack/blob/main/CHANGELOG_FORMAT.md))
- [ ] `README.md`
- [ ] Tests
- [ ] Storybook examples created/updated

### CODE REVIEW

**Review by giriawu — APPROVED** [review]

File: packages/bpk-component-switch/src/BpkSwitch.module.scss [hunk 1]
```diff
@@ -56,6 +56,11 @@ $border-width: $bpk-one-pixel-rem * 2;
   &__label {
     margin-right: bpk-spacing-md();
     line-height: $height;
+
+    @include bpk-rtl {
+      margin-right: 0;
```
Comment (dominicfraser) [inline packages/bpk-component-switch/src/BpkSwitch.module.scss:61]: Could we use `margin-inline-start` here?

cc: @LexSwed

Comment (dominicfraser) [reply packages/bpk-component-switch/src/BpkSwitch.module.scss:61]: (We haven't decided on its usage in Backpack, but in main web are moving away from bpk-rtl to towards native support where possible)

Comment (olliecurtis) [reply packages/bpk-component-switch/src/BpkSwitch.module.scss:61]: I think this would be safe to use here yes!

Comment (viktor-yang) [reply packages/bpk-component-switch/src/BpkSwitch.module.scss]: Make sense of that, thank you both.

**Review by olliecurtis — APPROVED** [review]

**Pull request merged**